### PR TITLE
Update README correct links

### PR DESCRIPTION
**Pull request by bd-g**

### CODE REVIEW

**Comment by bd-g** [comment]
Waiting for GH Secret to be added to repo before merging, so `publish` action can succeed.

File: podman/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ There are several [Podman Compose](https://github.com/containers/podman-compose)
 * [Podman Compose template](./podman-compose.license-proxy.yml) for a deployment with the [Deepgram License Proxy](https://developers.deepgram.com/docs/license-proxy)
     * [Associated Deepgram configuration files](../common/license_proxy_deploy/)
 ## Usage
-1. Read the [Deepgram self-hosted introduction](https://developers.deepgram.com/docs/self-hosted-introduction), then proceed to the [Docker/Podman](https://developers.deepgram.com/docs/dockerpodman) deployment path. Complete the series of guides until you have a deployment environment configured and are ready to begin the [Deploy Deepgram Services](https://developers.deepgram.com/docs/deploy-deepgram-services) guide.
+1. Read the [Deepgram self-hosted introduction](https://developers.deepgram.com/docs/self-hosted-introduction), then proceed to the [Podman/Podman](https://developers.deepgram.com/docs/podmanpodman) deployment path. Complete the series of guides until you have a deployment environment configured and are ready to begin the [Deploy Deepgram Services](https://developers.deepgram.com/docs/deploy-deepgram-services) guide.
```
[the remark this answers is not in the document]
Comment (bd-g) [reply podman/README.md:13]: Great catch! Gotta be more careful with `sed`

[the remark this answers is not in the document]
Comment (bd-g) [reply podman/README.md]: Resolved [here](https://github.com/deepgram/self-hosted-resources/pull/13/commits/eab95aa971fd37b17951c219ce36beec68df3e99)

File: charts/deepgram-self-hosted/README.md.gotmpl [hunk 2]
```diff
@@ -91,7 +91,11 @@ The Deepgram Helm chart takes references to two existing secrets - one containin
 
 Consult the [official Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/) for best practices on configuring Secrets for use in your cluster.
 
-## Troubleshooting
+## Getting Help
```
[the remark this answers is not in the document]
Comment (bd-g) [reply charts/deepgram-self-hosted/README.md.gotmpl:94]: The `README.md` is automatically generated from the template using [this tool](https://github.com/norwoodj/helm-docs).

The nice part about it is `helm-docs` automatically parses the `values.yaml` and `Chart.yaml` file to pull out version and configuration information and put it directly in the README. That way, there isn't duplication and we don't have drift.

So the README should never be manually edited. Any edits should happen in the template, and then run `helm-docs` to re-create the README.

[the remark this answers is not in the document]
Comment (bd-g) [reply charts/deepgram-self-hosted/README.md.gotmpl:94]: `helm-docs` is (quietly) called out in the [CONTRIBUTING page](https://github.com/deepgram/self-hosted-resources/blob/main/CONTRIBUTING.md#requirements)

[the remark this answers is not in the document]
Comment (jkroll-deepgram) [reply charts/deepgram-self-hosted/README.md.gotmpl:94]: Thanks for the explanation, helpful to know!

**Review by jkroll-deepgram — APPROVED** [review]

**Pull request merged**